vllm/model_executor/models/llama.py

```diff
@@ -179,6 +180,7 @@ def __init__(
             rope_scaling=rope_scaling,
             max_position_embeddings=max_position_embeddings,
             linear_method=linear_method,
+            bias=getattr(config, "bias", False),
```

Comment:
Shouldn't it be `attention_bias`? (Note they use the term in a different sense compared to the conventional one)

https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/configuration_llama.py#L161